common/list/lists-en.html

```diff
@@ -0,0 +1,583 @@
+---
+{
+	"title": "GCWeb Lists example",
```

Comment:
We can go simple, like how you identified those pages from the index.json-ld file

```suggestion
	"title": "Lists",
```